web/react/components/post_deleted_modal.jsx

```diff
@@ -2,13 +2,43 @@
 // See License.txt for license information.
 
 var UserStore = require('../stores/user_store.jsx');
+var AppDispatcher = require('../dispatcher/app_dispatcher.jsx');
+var Constants = require('../utils/constants.jsx');
+var ActionTypes = Constants.ActionTypes;
 
 export default class PostDeletedModal extends React.Component {
     constructor(props) {
         super(props);
 
+        this.handleClose = this.handleClose.bind(this);
+
         this.state = {};
     }
+    componentDidMount() {
+        const self = this;
+
+        $(React.findDOMNode(this.refs.modal)).on('hidden.bs.modal', () => {
+            self.handleClose();
```

Comment:
`this` gets automatically bound when using `=>` so no need for the 'self' declaration